# print("this is starting point of prgm")
# print("this is starting point of prgm")
# try:
#     print(x)
# except:
#     print("exception is  handled")  #nameerror exception
# print("this is end point of prgm")



#example2:
print("start of program")
try:
    print(10/0)
except  :
    print("exception handled")
print("end of program")
